packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -48,11 +78,23 @@ class PathBoundsMatcher extends Matcher {
   bool matches(covariant Path item, Map<dynamic, dynamic> matchState) {
     final Rect bounds = item.getBounds();
 
-    final List<Matcher?> matchers = <Matcher?> [rectMatcher, topMatcher, leftMatcher, rightMatcher, bottomMatcher];
```

Comment:
It looks like a code formatter was run in this file and it's hard to see what was actually changed, could you remove the formatting changes?